server/routes/contests/getContest.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const router = express.Router()
+const contests = require('../../models/contests')
+const middleware = require('../middlewares')
+
+router.get(
+  '/:contest_id',
+  middleware.verifyUserAccessToken,
```

Comment:
Users should be able to see this even without logging in.
So, I think it is better to add a boolean identifier in request body like `allowNoLogin` which can be used in `verifyUserAccessToken` middleware such that if it is true then do not respond with error continue by calling next function.

Reply:
We can make this a new feature later. For now this it is not needed